docs/js/isisDemos.js

```diff
@@ -0,0 +1,2270 @@
+
+    // static images
+    this.image = (this.options.staticImage) ? this.options.staticImage : false;
+
+    // layered images
+    if( this.options.hasOwnProperty('challenge') ) {
+	this.canvas = this.options.canvas || null;
+	this.ctx = this.options.context || null;
+	this.stage = this.options.stage || null;
+	this.challenge = this.options.challenge || null;
+	this.showOriginalId = this.options.target + '-showOriginal';
+	if( this.canvas != null && this.ctx != null && this.stage != null && this.challenge != null ) {
+	    this.turnOnFilters = true;
+	}
+    }
+
+    // build the console
+
+    // show the right console ( if the options are set to true )
+    if(this.options.showRightConsole) {
+	this.container.appendChild(this.displayRight);
+    }
+    // append the bottom of the console
+    this.container.appendChild(this.displayBottom);
+
+    // initialize slider element
+    if ( this.slider && this.options.showRightConsole ) {
+	this.sliderId = this.options.target + '-slider';
+	this.sliderValId = this.options.target + '-sliderVal';
+	this.initSlider('change');
+    }
+
+    // extra style fixes
+    if(this.options.bottomConsoleTop) { this.displayBottom.style.top = this.options.bottomConsoleTop; }
+
+    // pixel information - mouse position
+    this.mousePos = document.createElement('span');
+    this.mousePos.setAttribute('class', 'text');
+    this.displayBottom.appendChild(this.mousePos);
+    this.mousePos.innerHTML = "Pixel at: 0, 0";
+
+    // pixel information - DN at mouse position
+    var storedDnContainer = document.createElement('span');
+    storedDnContainer.setAttribute('class', 'text');
+    this.storedDn = document.createElement('input');
+    if(this.boxes.length > 0 || this.image) { 
+	storedDnContainer.innerHTML = "Stored DN: "; 
+	this.storedDn.value = 255;
+
+    } else { 
+	storedDnContainer.innerHTML = "RGB: "; 
+	this.storedDn.setAttribute('class', 'special-width');
+	this.storedDn.value = "255, 255, 255";
+    };
+    
+    storedDnContainer.appendChild(this.storedDn);
+    this.displayBottom.appendChild(storedDnContainer);
+    this.storedDn.readOnly = true;
+
+    // true DN
+    if( this.showDnMultiplier && this.options.showRightConsole ) {
+	this.dnMultiplierId = this.options.target + '-dnMultiplier';
+	this.dnMultiplierValId = this.options.target + '-dnMultiplierVal';
+	this.dnBaseId = this.options.target + '-dnBase';
+	this.dnBaseValId = this.options.target + '-dnBaseVal';
+
+	var trueDnContainer = document.createElement('span');
+	trueDnContainer.setAttribute('class', 'text');
+	var trueDn = document.createElement('input');
+	trueDnContainer.innerHTML = "True DN: ";
+	trueDn.value = 255;
+	trueDn.setAttribute('id', 'trueDn');
+	trueDnContainer.appendChild(trueDn);
+	this.trueDn = trueDn;
+	this.displayBottom.appendChild(trueDnContainer);
+	this.initSlider('base');
+	this.initSlider('multiplier');
+	
+	this.trueDn.readOnly = true;
+    }
```

Comment:
Maybe a bug in here? Multiplier demo seems to only how 255 for both stored and true DN

Reply:
Destripe and seam removal demos have the same problem, so maybe somewhere else?

Reply:
I'm pretty sure this is a cross-origin error, because the js on your localhost is trying to access images on another site.  The other site is the docs site, so this error should go away when that page is hosted in our bucket (same as the images).  If you want to test this, I think you can change the image srcs to local ones (lines 1775-2000 of isisDemos.js) for local testing.